packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -202,10 +202,16 @@ class PlatformPolyline {
   final bool consumesTapEvents;
   final int color;
   final bool geodesic;
+  // TODO(schectman): Convert field to enum.
```

Comment:
Please capture more context here (in case this last for a while), explaining that it currently relies on the Dart integer values in the platform interface package matching the integer values in the native SDK.